pyproject.toml

```diff
@@ -1,6 +1,86 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
```

Comment:
Did you like this build backend swap? I have one package that I started with `hatch` from the start and overall liked it. I'm in the process of switching a bunch of Django projects from `pip` to `uv` (so. much. faster.), but not usually swapping out the build backend

Reply:
I'm liking hatch now that I'm getting more used to it and more familiar with it - I'm sure I'm not using it to the full extent of what it's actually capable of. I've been tempted to switch to uv but haven't tried it yet, good to hear the endorsement.